### PR TITLE
feat: wishlist added

### PR DESCRIPTION
**Pull request by som749898**
Please review my component
Screenshot (87)

### CODE REVIEW

File: Wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    </header>
+    <div class="main-content flex flex-column horizontal-center vertical-center">
+      <h3 class="heading3 py-bottom-1x">My Wishlist</h3>
+      <div class="container2">
+        <div class="flex flex-wrap horizontal-center vertical-center">
+          <div class="card-container mg-bottom-1x">
+            <div class="card background-cover badge-container">
+              <span class="card-badge text-badge-position text-badge-style">New</span>
+              <i class="fa fa-times card-icon icon-badge-position"></i>
+              <img class="card-img" src="/assets/image/t-shirt-8.jpg" alt="image">
```
Comment (Aditi0505) [inline Wishlist/wishlist.html]: Please provide a better alt description. It is  a given that it is an image. You can write black_t-shirt-product or something more meaningful.

**Review by Aditi0505 — APPROVED** [review]
Left one suggestion. rest looks good to me.

**Comment by Digambar1411** [comment]
Good to go

**Comment by Digambar1411** [comment]
good to merge

**Review by Digambar1411 — APPROVED** [review]

File: Wishlist/wishlist.html (hunk 1 above)
Comment (som749898) [reply Wishlist/wishlist.html]: fixed it

**Pull request merged**